### ISSUE TITLE
How to Cite screenreader navigation fails

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4235]
From #4194, Sample 3 item 46 regarding the Clean theme using iOS Safari and voiceover :

> popup for citation doesn't disappear when focus shifts outside. For example, opening citation, then navigating by heading or link moves to new place in document, but popup remains on screen. Then you can navigate the whole original document, but if you go back to the Citation section, and then go down the links, having reached harvard-style-citation you are taken back into the model, no way to get to other subsequent items in the citation list. From the open citation pop up you cannot navigate forwards by heading, but you can navigate backwards, and then once in the main document, you can navigate at will.

When fixing this, check the 'how to cite' across other themes too.
